TiledLib/Tileset.cs

```diff
@@ -84,7 +90,7 @@ public class Tileset : ITileset, IXmlSerializable
 
         public static Tileset FromStream(System.IO.Stream stream)
         {
-            using (var reader = new System.IO.StreamReader(stream))
+            using (var reader = new System.IO.StreamReader(stream, System.Text.Encoding.UTF8, true, 1024, true))
```

Comment:
NB: This is a subtle, user visible, change. But now the behaviour matches between Tileset.FromStream and Map.FromStream.